website/releases/release-0.12.0.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Release 0.12.0"
+sidebar_position: 2
+layout: releases
+toc: true
+last_modified_at: 2022-08-17T10:30:00+05:30
+---
+# [Release 0.12.0](https://github.com/apache/hudi/releases/tag/release-0.12.0) ([docs](/docs/quick-start-guide))
+
+## Release Highlights
+
+### Presto-Hudi Connector
+
+Since version 0.275 of PrestoDB, users can now leverage native Hudi connector to query Hudi table. 
+It is on par with Hudi support in the Hive connector. To learn more about the usage of the connector, 
+please checkout [prestodb documentation](https://prestodb.io/docs/current/connector/hudi.html).
+
+### Archival Beyond Savepoint
+
+Users can now archive Hudi table beyond savepoint commit. Just enable `hoodie.archive.beyond.savepoint` write 
```

Comment:
Probably we can rephrase as below. 

```
Hudi supports savepoint and restore feature that users can use for disaster and recovery scenarios. More info can be found [here](https://hudi.apache.org/docs/disaster_recovery).  Until 0.12, archival for a given table will not make progress beyond first savepointed commit. But there has been ask from the community to relax this constraint so that we can retain some coarse grained commits in active timeline and execute point in time queries. So, with 0.12, users can now let archive proceed beyond savepoint commits by enabling `hoodie.archive.beyond.savepoint`. This unlocks new opportunities for Hudi users. For example, one can retain commits for years, by adding one savepoint per day for older commits (lets say > 30 days). And query hudi table using "as.of.instant" with any older savepointed commit. By this, hudi does not need to retain every commit in the active timeline for older commits. 
```

Let me know what you think.
```

Reply:
do retain the "note" section in the end.

Reply:
Sets the context very nicely. Sounds much better. Thanks.